internal/tspath/ignoredpaths.go

```diff
@@ -2,7 +2,7 @@ package tspath
 
 import "strings"
 
-var ignoredPaths = []string{"/node_modules/.", "/.git", "/.#"}
```

Comment:
This change should be redundant, afaik. `"/node_modules/."` already matches `"/node_modules/.pnpm"`. But also, the error just... isn't wrong? So long as these paths into the cache are the only paths we have available, we're generating bad specifiers that produce bad declaration files. Disabling the error conditionally is not the right fix - generating better module paths is.